diff --git a/src/lib/utils/onedrive-file-picker.ts b/src/lib/utils/onedrive-file-picker.ts
index 198a8bc..21f7af9 100644
--- a/src/lib/utils/onedrive-file-picker.ts
+++ b/src/lib/utils/onedrive-file-picker.ts
@@ -1,5 +1,6 @@
 import type { PopupRequest, PublicClientApplication } from '@azure/msal-browser';
 import { v4 as uuidv4 } from 'uuid';
+import { WEBUI_BASE_URL } from '$lib/constants';
 
 class OneDriveConfig {
 	private static instance: OneDriveConfig;
@@ -32,7 +33,7 @@ class OneDriveConfig {
 	}
 
 	private async getCredentials(): Promise<void> {
-		const response = await fetch('/api/config', {
+		const response = await fetch(`${WEBUI_BASE_URL}/api/config`, {
 			headers: {
 				'Content-Type': 'application/json'
 			},
